### DISCUSSION TITLE
Best way to share the files from the Torrent Container?

### DISCUSSION THREAD

**Discussion by Cookie1990** [tteck/Proxmox#1479] [Q&A]
Hello fellow selfhost admins!
So I deployed the qBittorrent LXC Container and it works great so far.
But how would I share my new linux ISO's now the most efficient way?

Keep it up, see ya!

**Comment by oOStroudyOo** [comment]
So you mean, you want to store your ISO files, and then seed them via qBittorent to other users?

**Reply by Cookie1990** [reply]
I mean:

What is the best way to share the ISO files I have downloaded with the rest of my network? 

I don't want to install another service in the container but keep the 1:1 principle of container:service.

Later, of course, I also want to push files to the torrent server so that they can be seeded via bittorrent.

mfg Florian
